### PR TITLE
docs(how-to guides): add documentation about controller evaluation

### PR DESCRIPTION
**Pull request by brkay54**
Signed-off-by: Berkay Karaman <berkay@leodrive.ai>

## Description

Related: #183 

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/how-to-guides/evaluating-controller-performance.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+![Start PlotJuggler](images/evaluating-controller-performance/start-plotjuggler.png)
+
+#### 6. Now, you can start to driving. You should see all the performance and driving variables in PlotJuggler
+
+![Controller Monitor](images/evaluating-controller-performance/controller-monitor.png)
+
+### After Driving
+
+#### 1. You can export the statistical output and all data to compate and later usage
```
Comment (kenji-miyake) [inline docs/how-to-guides/evaluating-controller-performance.md:59]: typo?

```suggestion
#### 1. You can export the statistical output and all data to compare and later usage
```

File: docs/how-to-guides/evaluating-controller-performance.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+![CVS Export](images/evaluating-controller-performance/export-cvs.png)
+
+- You can also export all data to later use. To investigate them again, after launch PlotJuggler, import the `.cvs` file from data section.
+
+![Import Data](images/evaluating-controller-performance/import-data.png)
+
+### Tips
+
+- You can plot the vehicle position. Select the two curve (keeping CTRL key pressed) and Drag & Drop them using the **RIGHT Mouse button**. Please visit the `Help -> Cheatsheet` in PlotJuggler to see more tips about it.
```
Comment (kenji-miyake) [inline docs/how-to-guides/evaluating-controller-performance.md:71]: FYI: Added to the dictionary.
https://github.com/tier4/autoware-spell-check-dict/pull/322

**Comment by kenji-miyake** [comment]
@TakaHoribe @rej55 Could you assign some developers from TIER IV's control team if necessary?

**Review by kenji-miyake — APPROVED** [review]
Looks great to me! Thank you for your work.
I feel the quality is good enough to be merged, but if necessary, you can wait for other developers' reviews because I'm not familiar with this area.

**Review by kenji-miyake — APPROVED** [review]

**Review by TakaHoribe — APPROVED** [review]

**Pull request merged**